cfgov/tccp/jinja2/tccp/card.html

```diff
@@ -94,23 +94,24 @@ <h2>Purchases</h2>
                     - No APRs: "The issuer did not submit APR information for this card"
             #}
             <dd>
-                {% if card.purchase_apr_min
-                    or card.purchase_apr_max
-                    or card.purchase_apr_median
+                {% if card.purchase_apr_min is not none
+                    or card.purchase_apr_max is not none
+                    or card.purchase_apr_median is not none
                 %}
                     {% set purchase_apr_range =
                         apr_range(card.purchase_apr_min, card.purchase_apr_max) if
-                        card.purchase_apr_min and card.purchase_apr_max
+                        card.purchase_apr_min is not none
+                        and card.purchase_apr_max is not none
                     %}
                     {% set purchase_apr_median = apr(card.purchase_apr_median) if
-                        card.purchase_apr_median
+                        card.purchase_apr_median is not none
                     %}
                     {{ purchase_apr_range if purchase_apr_range }}
                     {{ '(' if purchase_apr_median
                         and purchase_apr_range
                         and purchase_apr_median != purchase_apr_range
                     }}
-                    {{- purchase_apr_median + ' median APR' if purchase_apr_median
+                    {{- purchase_apr_median ~ ' median APR' if purchase_apr_median
```

Comment:
@chosak, sorry, I named `purchase_apr_median` a little confusingly. It's a variable that's set earlier:

```jinja2
{% set purchase_apr_median = apr(card.purchase_apr_median) if
    card.purchase_apr_median is not none
%}
```
So I think it will either be not set (if `card.purchase_apr_median` is `none`) or be a string like `5.9%`. `purchase_apr_median` will probably never be `0%`, but the similar `intro_apr_median`  might be. But given that both will be either not set or a string, is it OK to leave the condition as just `if purchase_apr_median`? Should I change the variable name to be more descriptive and less similar to the `card.purchase_apr_median` property, like `purchase_apr_median_percentage` or something?

Reply:
Oh, no, my bad for misreading. This is great.